### PR TITLE
Issue67 (Move the `max_delay` argument of `setup()` into `extra_params`)

### PR DESCRIPTION
**Pull request by jgonin**

### CODE REVIEW

**Comment by apdavison** [comment]
You will also need to update the definition of `setup()` in `common/control.py`

File: examples/IF_and_HH_responses_to_injection.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+# encoding: utf-8
```
Comment (apdavison) [inline examples/IF_and_HH_responses_to_injection.py]: This file should not be part of this pull request. You will need to remove it from your branch.

Comment (jgonin) [inline examples/IF_and_HH_responses_to_injection.py]: Oh yes ! Sorry for that, I do not know what happened here. Thanks you.

**Comment by jgonin** [comment]
Do not accept this pull request yet, I made a mistake...
